experiments/data/prepare_chemrxiv.py

```diff
@@ -0,0 +1,43 @@
+"""
+Preparing chemrxiv dataset as per GPT-NeoX guidelines
+NOTE this needs to be run from the root of this repository directory
+
+Example usage:
+    python experiments/chem_data_prep.py /fsx/proj-chemnlp/data/ chemnlp/gpt-neox/
+"""
+import argparse
+import os
+
+import datasets
+import jsonlines
+
+DATASET = "marianna13/chemrxiv"
+GPT_NEOX_KEY = "text"
+
+if __name__ == "__main__":
+    # parse args
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "save_dir", help="Where you want to store the prepared dataset."
+    )
+    parser.add_argument(
+        "gptneox_dir", help="Where you can find the GPT-NeoX repository."
+    )
+    args = parser.parse_args()
+
```

Comment:
Minor point, should we agree on what framework we use for CLIs? (`click`, `fire`, simple `argparse`)

Reply:
Ye I agree, I've used `click` before but don't really have any strong preference tbh